components/try.tsx

```diff
@@ -13,28 +16,63 @@ export default function TryPear() {
         </div>
 
         {/* Content area */}
-        <div className="relative pb-10 pt-32 md:pb-16 md:pt-40">
+        <div className="relative pb-2 pt-32 md:pb-4 md:pt-40">
           {/* Section header */}
           <div className="mx-auto max-w-3xl pb-12 text-center md:pb-16">
-            <h1 className="h1 mb-4" data-aos="fade-up">
-              Try PearAI For Free
+            <h1 className="mb-4 text-4xl font-bold" data-aos="fade-up">
+              Interested in contributing to PearAI?
             </h1>
+            <p
+              className="text-secondary-400 mb-4 text-sm sm:text-lg"
+              data-aos="fade-up"
+              data-aos-delay="200"
+            >
+              Pear is built by a large community of developers. If you have
+              questions, or would like to discuss, you can join our{" "}
+              <Link
+                href="https://discord.gg/7QMraJUsQt"
+                className="hover:text-primary-800 text-primary-700"
+              >
+                Discord
+              </Link>{" "}
+              and talk to us directly! 🗣️
+            </p>
+            <p
+              className="text-secondary-400 mb-8 text-sm sm:text-lg"
+              data-aos="fade-up"
+              data-aos-delay="200"
+            >
+              Help the community out by giving{" "}
+              <Link
+                href="https://github.com/trypear/pearai-app"
+                className="hover:text-primary-800 text-primary-700"
+              >
+                the repo
+              </Link>{" "}
+              a star! 🤩
+            </p>
             <div className="mx-auto max-w-xs sm:flex sm:max-w-none sm:justify-center">
               <div data-aos="fade-up" data-aos-delay="400">
-                <a
-                  className="btn mb-4 w-full bg-primary-500 text-white-main hover:bg-primary-600 sm:mb-0 sm:w-auto"
-                  href="https://forms.gle/171UyimgQJhEJbhU7"
+                <Button
+                  asChild
+                  size={"lg"}
+                  className="hover:bg-primary-800 mb-4 w-full rounded-full bg-primary-700 text-white-main sm:mb-0 sm:w-auto"
```

Comment:
```suggestion
                  className="hover:bg-primary-800 w-full rounded-full bg-primary-700 text-white-main hover:shadow-sm sm:w-auto"
```
button styling used from hero